### ISSUE TITLE
ALT+PgUp and ALT+ PgDn work fine, until you release the ALT key (Windows)

### ISSUE THREAD

**Issue by Toneplay** [zim-desktop-wiki/zim-desktop-wiki#1294]
Version 0.73.3 for Windows.

If I press and hold ALT, then press PgDn or PgUp several times to go to prior and next pages, when I finally release the ALT key, a spurious, unpredictable, sometimes unprintable (garbage) character is inserted into the page I'm on.

It would be nice if there were no characters inserted when I release the ALT key.

**Comment by jaap-karssenberg** [comment]
Cannot reproduce this issue on my system. Do you use a non-english keyboard
layout, or do you have any special keyboard shortcuts setup that might
interfere ?

-- Jaap

**Comment by Toneplay** [comment]
I stopped all other programs and still had the problem. THEN I tried the _other_ PgDn and PgUp keys. . .

The problem is caused by using PgUp and PgDn _on the numeric keypad_ while NumLock is off. Using the native PgUp and PgDn keys do not cause the problem.

(Unfortunately, on my laptop keyboard, the native PgUp and PgDn keys are further away than the 9 and 3 on the numeric keypad. But I'll use them, for now!)
